blakcomp/function.c

```diff
@@ -85,7 +85,8 @@ function_type Functions[] = {
 {"RecycleUser",         RECYCLEUSER,     AEXPRESSION,   ANONE},
 {"MinigameNumberToString", MINIGAMENUMBERTOSTRING,     AEXPRESSION,   AEXPRESSION,   ANONE},
 {"MinigameStringToNumber", MINIGAMESTRINGTONUMBER,     AEXPRESSION,   ANONE},
-   };
+{"GetServerConfigValue", GETSERVERCONFIGVALUE,  C_STRING,   ANONE},
```

Comment:
Should I be using `AEXPRESSION` here rather than a `C_STRING`? Or `C_NUMBER`?

Reply:
Yes, AEXPRESSION.